.gitignore

```diff
@@ -17,7 +17,6 @@ target/*
 .idea/
 .lein-repl-history
 example.rb
-Dockerfile
```

Comment:
OK, I done it like this because I'm Windows .. of course this is temporary .. just wanted to send the version that I know for sure working .. But I'll make that `make` work in my OS ..